includes/class-wcs-blocks-integration.php

```diff
@@ -97,4 +98,35 @@ protected function get_file_version( $file ) {
 		}
 		return \WC_Subscriptions_Core_Plugin::instance()->get_library_version();
 	}
+
+	/**
+	 * Fetches the place order button text if it has been overridden by one of Woo Subscription's methods.
+	 *
+	 * @return string|null The overridden place order button text or null if it hasn't been overridden.
+	 */
+	protected function get_place_order_button_text_override() {
+		$default           = null;
+		$order_button_text = $default;
+
+		// Check if any of our button text override functions (hooked onto 'woocommerce_order_button_text') change the default text.
+		$callbacks = [
```

Comment:
@james-allan are we missing the resubscribe callback from this list?

| shortcode checkout | block checkout |
|--------|--------|
| ![image](https://github.com/Automattic/woocommerce-subscriptions-core/assets/2275145/5dcff8d5-4385-4d24-a753-509c3b942054) | ![image](https://github.com/Automattic/woocommerce-subscriptions-core/assets/2275145/b10538db-a7bf-4aff-98b8-61a3b140a46b) |

Reply:
Yep. Good catch. I missed this by only searching for the hook name but this is inherited by the cart renewal hooking into it. Fixed in https://github.com/Automattic/woocommerce-subscriptions-core/pull/587/commits/8d1f929d157822e2b645981c76f1d93132754fdd